api/CredentialsContainer.json

```diff
@@ -32,7 +32,7 @@
             "version_added": "13"
           },
           "safari_ios": {
-            "version_added": "13.3"
+            "version_added": "13"
```

Comment:
OK, so `CredentialsContainer` was supported earlier in Safari iOS than the previous recorded, and that matches the Safari macOS data so is very plausible.